main.py

```diff
@@ -17,10 +17,10 @@
     # separator = args["sep"]
 
     detector = Detector(args['train'], args['test'], delimiter=args['sep'])
-    detector.add_checks(SimpleCheck())
```

Comment:
I think this file should not be in the repository. Can you delete it and write it on .gitignore?

Reply:
it already is in the gitignore... changed its position, maybe it works now